paper.tex

```diff
@@ -476,137 +476,156 @@ \section*{Appendix}
 
 Population sizes are given as numbers of individuals, and details
 such as ploidy levels are considered external to the model.
-We therefore focus on the number of individuals as opposed to the number of genomes.
+We therefore focus on the number of individuals as opposed
+to the number of genome copies.
 Sizes and mating system details are specified for each population within
 epochs.
 Epochs are contiguous time intervals that define
 the existence interval of the population. Each epoch specifies the population size
 over that interval, which can be a constant value or function defined by start
 and end sizes that must remain positive.
 
-\krtcomment{The paragraph(s) below subsume(s)/replace(s) some of what appears in later pargraphs.
-	I'm not making an attempt yet to revise that later content--I think other authors should
-	weigh in.}
+%\krtcomment{The paragraph(s) below subsume(s)/replace(s) some of what appears in later pargraphs.
+%	I'm not making an attempt yet to revise that later content--I think other authors should
+%	weigh in.}
 
-\krtcomment{Aaron -- I have perhaps over-simplified the description of marginal allele frequency dynamics?}
+%\krtcomment{Aaron -- I have perhaps over-simplified the description of marginal allele frequency dynamics?}
 
 Within a population, we implicitly assume that allele frequency dynamics can
-be described by the Wright-Fisher model.  Briefly, generations are non-overlapping (all parents
-reproduce and die simultaneously) and the expected frequency (at birth) of the $i^{th}$ allele currently at frequency $p_i$
-one generation in the next generation is $p_iw_i/\bar{w}$, where $w_i$ and $\bar{w}$ are the marginal and
-mean fitnesses, respectively, properly weighted according to ancestry proportions. In this framework,
-a forward-time simulation of finite populations is equivalent to multinomial sampling of allele frequencies each
-generation (\citet[][pp 29-31]{burger2000-ul}, \citet[][pp 179-181]{crowkimura1970})
-and a backwards-time (coalescent) simulation follows the approximations described in \cite[][chapter 3]{wakeley2008-hd}.
-Further, this model assumes "soft" selection \citep{christiansen1975hard}, meaning that the dynamics of population sizes
+be described by the Wright-Fisher model.
+Briefly, generations are non-overlapping (all parents
+reproduce and die simultaneously) and the expected frequency (at birth)
+of the $i^{th}$ allele currently at frequency $p_i$
+one generation in the next generation is $p_iw_i/\bar{w}$,
+where $w_i$ and $\bar{w}$ are the marginal and
+mean fitnesses, respectively, properly weighted according to ancestry proportions.
+In this framework,
+a forward-time simulation of finite populations is equivalent
+to multinomial sampling of allele frequencies each
+generation (\citet[][pp 29-31]{burger2000-ul}, \citet[][pp 179-181]{crowkimura1970}),
+and a backwards-time (coalescent) simulation follows the approximations
+described in \cite[][chapter 3]{wakeley2008-hd}.
```

Comment:
Wakeley is a fine ref here, but Hudson and especially Tajima are probably worth including.  Also, and practically, it may be harder for future readers to get a hold of Wakeley's book.

Reply:
Hudson (2002) and Tajima (?) - which did you have in mind here?

Reply:
1983 for both.